spec/features/applications/new_spec.rb

```diff
@@ -19,4 +19,12 @@
     expect(page).to have_content('In Progress')
     expect(page).to have_content('Lots of love!')
   end
+
+  it "sends error message when form isn't complete" do
+    visit "/applications/new"
+
+    click_on("Submit")
+
+    expect(page).to have_content("Error submitting request, please fill in all fields")
+  end
```

Comment:
I think we want it to expect to have_button('Submit') here as well to make sure the button is still there.